### PR TITLE
Disregard.

### PR DESCRIPTION
**Pull request by briantoth**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/MageRings/draftomatic/pull/34" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -4,25 +4,7 @@
...
-
-# About
-
-Draftomatic seeks to provide a client-server architecture for swiss pairings.  Ideally this will disassociate the relatively complicated logic of pairings players from the acts of displaying the
-pairings or recording results.
-
-#Developing
-
-1. Run `./gradlew eclipse`.  This will fetch the java dependencies
-and set up an eclipse project.  Note that if you want to launch the
```
Comment (briantoth) [inline README.md:21]: ok

File: README.md [hunk 2]
```diff
@@ -4,25 +4,7 @@
...
-        ./gradlew run
-
-## View the application at:
-        http://localhost:8000/magic/
-
-# About
-
-Draftomatic seeks to provide a client-server architecture for swiss pairings.  Ideally this will disassociate the relatively complicated logic of pairings players from the acts of displaying the
-pairings or recording results.
-
```
Comment (briantoth) [inline README.md:17]: new comment

Comment (briantoth) [inline README.md:17]: test

File: README.md [hunk 3]
```diff
@@ -4,25 +4,6 @@
...
 
-## Run:
-        ./gradlew run
-
-## View the application at:
-        http://localhost:8000/magic/
-
-# About
-
-Draftomatic seeks to provide a client-server architecture for swiss pairings.  Ideally this will disassociate the relatively complicated logic of pairings players from the acts of displaying the
```
Comment (briantoth) [inline README.md:15]: test

**Pull request closed without merging**